docs/how-to/ChIP_seq_genomic_ranges_tutorial.ipynb

```diff
@@ -0,0 +1,2733 @@
+{
```

Comment:
<p>maybe have the column names formatted like <code>chrom</code>, <code>start</code>,<code>end</code>. Also very minor but I think it's ascribed instead of asscribed      </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/scverse/genomic-features/pull/65/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>